ErizoClient/ECRoom.h

```diff
@@ -165,6 +165,16 @@ typedef NS_ENUM(NSInteger, ECRoomErrorStatus) {
  */
 - (void)room:(ECRoom *)room didRemovedStreamId:(NSString *)streamId;
 
+/**
+ Fired when a data stream is received.
+ 
+ @param room Instance of the room where event happen.
+ @param stream The id received from.
+ @param data stream message received.
+ 
+ */
+- (void)room:(ECRoom *)room fromStreamId:(NSString *)streamId receivedDataStream:(NSDictionary *)dataStream;
```

Comment:
P2P support has been added recently, wonder if this will work with that since there is not only streamId in play, it also uses peerId to find out to which stream is talking to.